execution_engine/src/core/runtime/externals.rs

```diff
@@ -992,10 +996,10 @@ where
                     _,
                 ) = Args::parse(args)?;
                 self.charge_host_function_call(
-                    &host_function_costs.dictionary_get,
+                    &host_function_costs.read_value_local,
                     [
-                        uref_ptr,
-                        uref_size,
+                        // uref_ptr,
+                        // uref_size,
```

Comment:
Are these a `todo` perhaps or are they intentionally left commented-out as a note? (If intentional, perhaps we could add a note as to why they are not included in the call to `charge_host_function_call()`).

Reply:
Removed - when we'll change host function costs to increase arity compiler will complain here